contrib/charts/dragonfly/ci/prometheusrules-values.yaml

```diff
@@ -0,0 +1,15 @@
+serviceMonitor:
+  enabled: true
+prometheusRule:
+  enabled: true
```

Comment:
Can you also add the namespace to verify the same? 🤔

Reply:
Done! Thanks! (It still doesn't update the golden file as the main constraint ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) doesn't return true on my local)